docs/source/DownloadableImages.rst

```diff
@@ -44,7 +44,7 @@ JeOS image
 
 You can find the JeOS images currently available here:
 
-https://assets-avocadoproject.rhcloud.com/static/jeos-23-64.qcow2.7z
+https://assets-avocadoproject.rhcloud.com/static/jeos-23-64.qcow2.xz
 
 https://assets-avocadoproject.rhcloud.com/static/SHA1SUM_JEOS23
```

Comment:
This should become, for a brief transition period:

https://assets-avocadoproject.rhcloud.com/static/SHA1SUM_JEOS23_XZ

The idea is to support users with and without this change for a few days (both the .7z and .xz images will be there too).